content/open-source/wolfi/hello-wolfi.md

```diff
@@ -121,19 +121,17 @@ With both the PHP and Composer dependencies in place, you can now build the appl
 On Linux systems, use the following command:
 
 ```shell
-docker run --privileged --rm -v "${PWD}":/work -- \
-  cgr.dev/chainguard/melange build melange-app.yaml \
-  --arch x86_64 \
-  --signing-key melange.rsa --keyring-append melange.rsa.pub
+docker run --privileged --rm -v ${PWD}:/work cgr.dev/chainguard/melange \
```

Comment:
This command fails for me, I start getting this error:

```
. . .

2024/11/22 20:03:35 INFO running step "Build Minicli application"
2024/11/22 20:03:35 INFO PHP's phar extension is missing. Composer requires it to run. Enable the extension or recompile php without --disable-phar then try again. name="Build Minicli application"
2024/11/22 20:03:35 INFO deleting guest dir /tmp/melange-guest-1475690852
2024/11/22 20:03:35 INFO deleting workspace dir /tmp/melange-workspace-3361137084
2024/11/22 20:03:35 INFO removing image path /tmp/melange-guest-2907343915
2024/11/22 20:03:35 ERRO failed to build package: unable to run package hello-minicli pipeline: unable to run pipeline: exit status 1
```

There's always a chance this is just something I've messed up on my side, let me know if I can provide any more info on what's going on here

Reply:
FYI the remaining commands all fail after this as well

Reply:
Ok, let me check everything again, it seems some extension is missing